elao.app/Makefile.dist

```diff
@@ -17,49 +17,49 @@ install:
 	# # Doctrine
 	# bin/console doctrine:database:create --ansi --if-not-exists
 	# # Npm
-	# npm install --color=always
+	# npm install --color
```

Comment:
J'ai pas l'impression que le comportement ait changé -> https://docs.npmjs.com/cli/v7/using-npm/config#color

Ce qu'on veut, c'est qu'il y ait les couleurs, même si pas de tty, or si j'en crois la doc `If true, then only prints color codes for tty file descriptors.`

T'es sur de ton coup ?

Reply:
Non ^^ j'ai changé les `--color=always` partout par `--color` parce que ça ne marchait pas sur le watch. Mais c'est peut être que sur les call à webpack.

```
[webpack-cli] Error: Unknown option '--color=always'
```

Reply:
après, je ne vois pas d'option `--color` always ou pas dans le help de npm.

```
 npm --help

Usage: npm <command>

where <command> is one of:
    access, adduser, audit, bin, bugs, c, cache, ci, cit,
    clean-install, clean-install-test, completion, config,
    create, ddp, dedupe, deprecate, dist-tag, docs, doctor,
    edit, explore, fund, get, help, help-search, hook, i, init,
    install, install-ci-test, install-test, it, link, list, ln,
    login, logout, ls, org, outdated, owner, pack, ping, prefix,
    profile, prune, publish, rb, rebuild, repo, restart, root,
    run, run-script, s, se, search, set, shrinkwrap, star,
    stars, start, stop, t, team, test, token, tst, un,
    uninstall, unpublish, unstar, up, update, v, version, view,
    whoami

npm <command> -h  quick help on <command>
npm -l            display full usage info
npm help <term>   search for help on <term>
npm help npm      involved overview

Specify configs in the ini-formatted file:
    /home/vagrant/.npmrc
or on the command line via: npm <command> --key value
Config info can be viewed via: npm help config
```

C'est npm v6 chez moi, pas v7.

Reply:
Comportement à priori identique sur la v6 -> https://docs.npmjs.com/cli/v6/using-npm/config#color

(oublie le help de npm, il est la pour faire joli :) )

Pour les call à webpack tu passes par npx ?

Reply:
Pour les assets j'ai juste décommenté ce qu'il l'était dans le Makefile, donc oui, `npx encore ...`